src/pages/public_registration/LogPanel.tsx

```diff
@@ -44,22 +46,39 @@ export const LogPanel = ({ tickets, registration }: LogPanelProps) => {
     staleTime: Infinity,
     gcTime: 1_800_000, // 30 minutes
   });
-
-  const userQuery = useQuery({
-    enabled: !!resourceOwnerId,
-    queryKey: ['user', resourceOwnerId],
-    queryFn: resourceOwnerId ? () => fetchUser(resourceOwnerId) : undefined,
-    retry: 0,
-    staleTime: Infinity,
-    gcTime: 1_800_000, // 30 minutes
-  });
+  const organizationAcronym = organizationQuery.isPending
+    ? ''
+    : organizationQuery.data
+      ? organizationQuery.data?.acronym
+      : t('common.unknown');
 
   const logs: LogItem[] = [];
 
+  if (isImported) {
+    registration.importDetails?.map((importDetail) => {
+      const registrationImported: LogItem = {
+        modifiedDate: importDetail.importDate,
+        title: `${t('common.importedFrom')} ${importDetail.source}`,
```

Comment:
Kunne hatt en key som var `"imported_from": "Importert fra {{source}}"` også, for å unngå å slå sammen. Men det er bare gitt at "Importert fra" ikke skal vises for seg selv også da. Trenger ikke endre dette, your call!

Reply:
Godt forslag!